apps/antalmanac/src/components/RightPane/SectionTable/SectionTableBody.tsx

```diff
@@ -403,13 +517,6 @@ const SectionTableBody = withStyles(styles)((props: SectionTableBodyProps) => {
 
     // Stable references to event listeners will synchronize React state with the store.
 
-    const updateColumns = useCallback(
-        (newActiveColumns: SectionTableColumn[]) => {
-            setColumns(newActiveColumns);
-        },
-        [setColumns]
-    );
-
     const updateHighlight = useCallback(() => {
         setAddedCourse(AppStore.getAddedSectionCodes().has(`${section.sectionCode} ${term}`));
     }, [setAddedCourse]);
```

Comment:
We can remove this from the dependency array.

```suggestion
    }, []);
```